### ISSUE TITLE
Package missing in class reference

### ISSUE THREAD

**Issue by rjernigan** [stoicflame/enunciate#244]
When I run `mvn enunciate:docs`, I'm getting errors like the following:

```
error: cannot find symbol
  private java.util.Map<java.lang.String, com.codahale.metrics.health.Result> _fields;

error: cannot find symbol
  public java.util.Map<java.lang.String, com.codahale.metrics.health.Result> getFields() {
```

The file for this class can be seen here in the CodaHale project:
[metrics-adapter/src/main/java/com/codahale/metrics/health/HealthCheck.java](https://github.com/dropwizard/metrics/blob/796663609f310888240cc8afb58f75396f8391d2/metrics-adapter/src/main/java/com/codahale/metrics/health/HealthCheck.java)

It seems like the class is being referenced as `com.codahale.metrics.health.Result` instead of `com.codahale.metrics.health.HealthCheck.Result`

**Comment by stoicflame** [comment]
Enunciate flattens out the inner classes in the generated client-side code. But that doesn't explain why the compile is failing.

Do you have a branch of the project somewhere I can check out to reproduce the issue? Or a patch?

**Comment by rjernigan** [comment]
Unfortunately I'm not able to provide the project since it's private. In my project I have a method like so:

```java
public Map<String, HealthCheck.Result> getResults() {
	return results;
}
```

When the enunciate:docs goal is run, the following output is generated into the target directory:

```java
  /**
   * (no documentation provided)
   */
  @JsonProperty( value = "results", required = false )
  public java.util.Map<java.lang.String, com.codahale.metrics.health.Result> getResults() {
    return this._results;
  }
```

Where the incorrect package name is being generated from the CodaHale project.

I can try to reproduce the issue in a smaller non related project if that would help.

**Comment by rjernigan** [comment]
I've updated to 2.2.0 since I've posted this issue and it seems to have been resolved.  If I see it again I will open with more info. Thanks!

**Comment by stoicflame** [comment]
Fixed at c380e69.

**Comment by stoicflame** [comment]
[Enunciate 2.3 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.3.0).
